app/views/admin/profile_fields/_group_form.erb

```diff
@@ -0,0 +1,36 @@
+<div>
```

Comment:
The group form is a partial that contains a select to choose an existing group and an input to create a new group. There's some JS in the file that hides the one when the other is selected.